src/components/CollectionsAndPalettes/common/PaletteCard/PaletteCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { PaletteT } from "constants/types";
+import * as React from "react";
+import styles from "../../common/ItemCard/ItemCard.module.scss";
+import Contributors from "../Contributors/Contributors";
+import cn from "classnames";
+
+interface Props {
+  palette: PaletteT;
+}
+
+export default class PaletteCard extends React.Component<Props, any> {
+  renderVisibility = () => {
+    const { palette } = this.props;
+    if (palette.isPublic) {
+      return `Shared (${palette.collaborators.length} people)`;
+    }
+    return `Private`;
+  };
+
+  render() {
+    console.log(this.props);
```

Comment:
We should avoid leaving logs in render, as they usually end up spamming the console.
Ideally, at this stage we would only have logs for context or API-related issues so that we can debug stuff on the deployed app if needed.